test/resources/verify.go

```diff
@@ -212,7 +212,7 @@ func KSOperatorCRDelete(t *testing.T, clients *test.Clients, names test.Resource
 
 	// verify all but the CRD's and the Namespace are gone
 	for _, u := range m.Filter(mf.NoCRDs, mf.None(mf.ByKind("Namespace"))).Resources() {
-		if _, err := m.Client.Get(&u); !apierrs.IsNotFound(err) {
+		if _, err := m.Client.Get(&u); err != nil && !apierrs.IsNotFound(err) {
```

Comment:
But we're expecting an error here, right? If we can successfully `Get` something, the test should fail, I think?

Reply:
Ah, yes. Sorry, I misunderstood the meaning of this test. I close this. Thank you.

Reply:
Besides, `IsNotFound(err)` will happily return false for a nil `err`